File: Software-Tecnologies/CSharp/C# Skeleton TODO/TodoList/Models/Task.cs
using System.ComponentModel.DataAnnotations;

namespace TodoList.Models
{
    public class Task
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [StringLength(255)]
        public string Title { get; set; }

        [Required]
        public string Comments { get; set; }
    }
}

File: Software-Tecnologies/Java/Skeleton TODO/src/main/java/todolist/entity/Task.java
package todolist.entity;

import org.hibernate.validator.constraints.NotBlank;

import javax.persistence.*;

@Entity
@Table(name = "tasks")
public class Task {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @NotBlank
    private String title;

   @NotBlank
    private String comments;

    public Task(String title, String comments) {
        this.title = title;
        this.comments = comments;
    }

    public Task(){ }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getComments() {
        return comments;
    }

    public void setComments(String comments) {
        this.comments = comments;
    }
}
